xbmc/interfaces/Builtins.cpp

```diff
@@ -166,6 +167,7 @@ const BUILT_IN commands[] = {
   { "AlarmClock",                 true,   "Prompt for a length of time and start an alarm clock" },
   { "CancelAlarm",                true,   "Cancels an alarm" },
   { "Action",                     true,   "Executes an action for the active window (same as in keymap)" },
+  { "DialogOK",                   true,   "Shows an OK dialog on screen, specify header, then message." },
```

Comment:
"DialogOK" is used in a lot of APIs in general and skinners are familiar with it because of the dialog file name, so I dont think that the cmd name is not too bad
"Alert" would be pretty close to the already existing "AlarmClock".
@ronie , your opinion?